### ISSUE TITLE
[Bug]: PeoplePickerItemSuggestion is not accessible in Aquatic/Desert mode (contrast ratio)

### ISSUE THREAD

**Issue by vlumao** [microsoft/fluentui#34517]
### Package

common-styles

### Package version

8.118.2

### React version

17.0.2

### Environment

```shell
System:
    OS: Windows 11 10.0.26100
    CPU: (12) x64 13th Gen Intel(R) Core(TM) i7-1355U
    Memory: 2.02 GB / 15.68 GB
  Browsers:
    Edge: Chromium (135.0.3179.66)
  npmPackages:
    @fluentui/merge-styles: 8.6.4 => 8.6.4
    @fluentui/react: 8.118.2 => 8.118.2
    @fluentui/react-accordion: 9.4.3 => 9.4.3
    @fluentui/react-avatar: 9.6.32 => 9.6.32
    @fluentui/react-badge: 9.2.40 => 9.2.40
    @fluentui/react-button: 9.3.95 => 9.3.95
    @fluentui/react-card: 9.0.86 => 9.0.86
    @fluentui/react-carousel: 9.3.1 => 9.3.1
    @fluentui/react-charting: 5.21.7 => 5.21.7
    @fluentui/react-checkbox: 9.2.31 => 9.2.31
    @fluentui/react-combobox: 9.13.1 => 9.13.1
    @fluentui/react-dialog: 9.11.5 => 9.11.5
    @fluentui/react-divider: 9.2.72 => 9.2.72
    @fluentui/react-drawer: 9.5.5 => 9.5.5
    @fluentui/react-field: 9.2.0 => 9.2.0
    @fluentui/react-hooks: 8.8.1 => 8.8.1
    @fluentui/react-icons: 2.0.245 => 2.0.245
    @fluentui/react-image: 9.1.70 => 9.1.70
    @fluentui/react-infolabel: 9.0.56 => 9.0.56
    @fluentui/react-input: 9.4.82 => 9.4.82
    @fluentui/react-jsx-runtime: 9.0.53 => 9.0.53
    @fluentui/react-link: 9.2.27 => 9.2.27
    @fluentui/react-menu: 9.14.10 => 9.14.10
    @fluentui/react-message-bar: 9.2.5 => 9.2.5
    @fluentui/react-persona: 9.2.91 => 9.2.91
    @fluentui/react-popover: 9.9.31 => 9.9.31
    @fluentui/react-progress: 9.1.81 => 9.1.81
    @fluentui/react-search: 9.0.11 => 9.0.11
    @fluentui/react-select: 9.1.81 => 9.1.81
    @fluentui/react-shared-contexts: 9.23.1 => 9.23.1
    @fluentui/react-skeleton: 9.1.9 => 9.1.9
    @fluentui/react-slider: 9.3.0 => 9.3.0
    @fluentui/react-spinner: 9.4.11 => 9.4.11
    @fluentui/react-switch: 9.1.88 => 9.1.88
    @fluentui/react-table: 9.15.10 => 9.15.10
    @fluentui/react-tabs: 9.6.0 => 9.6.0
    @fluentui/react-tabster: 9.24.0 => 9.24.0
    @fluentui/react-tag-picker: 9.2.2 => 9.2.2
    @fluentui/react-tags: 9.3.11 => 9.3.11
    @fluentui/react-teaching-popover: 9.1.20 => 9.1.20
    @fluentui/react-text: 9.4.22 => 9.4.22
    @fluentui/react-textarea: 9.3.82 => 9.3.82
    @fluentui/react-theme: 9.1.24 => 9.1.24
    @fluentui/react-tooltip: 9.4.33 => 9.4.33
    @fluentui/react-tree: 9.7.4 => 9.7.4
    @fluentui/react-utilities: 9.18.23 => 9.18.23
    @fluentui/style-utilities: 8.10.9 => 8.10.9
    @types/react: 17.0.47 => 17.0.47
    @types/react-dom: 17.0.17 => 17.0.17
    react: 17.0.2 => 17.0.2
    react-dom: 17.0.2 => 17.0.2
```

### Current Behavior

For the PeoplePicker issue, it only happens when **we just use PeoplePickerItemSuggestion component**. When we actually hover on it, we can see this issue. I attached more screenshots here.

Our site:

![Image](https://github.com/user-attachments/assets/d9c897c5-b148-41dc-ba6f-e45d37ac5a0e)

![Image](https://github.com/user-attachments/assets/20c71d18-688e-4d9a-8b5d-31083ef8f980)

FluentUI storybook

![Image](https://github.com/user-attachments/assets/97d8cf51-d854-46b8-b36c-7a770f9a8ab0)

![Image](https://github.com/user-attachments/assets/135437e2-03d0-4a6d-ab8c-5f82e4456ab2)

### Expected Behavior

When user hovers over the people Picker, the low contrast makes it difficult to see. There should be a more clear contrast ratio. Currently, the contrast ratio for hovering text is 1.169:1 which is less than 4.5:1 .

### Reproduction

https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker

### Steps to reproduce

Go to site above: https://developer.microsoft.com/en-us/fluentui#/controls/web/peoplepicker
Go to Normal People Picker
Change display to aquatic mode
Apply css: hover
see issue

### Are you reporting an Accessibility issue?

yes

### Suggested severity

Urgent - No workaround and Products/sites are affected

### Are you willing to submit a PR to fix?

yes

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by vlumao** [comment]
I have to manually apply hover css to the elements on storybook site to see the issue but we can see this issue when we actually hover on it on our site since we use PeoplePickerItemSuggestion directly. Please let me know if more information is needed to triage this issue. Thanks!

**Comment by smhigley** [comment]
@vlumao could you provide a repro using the "Export to Codepen" button that shows the issue without needing to selectively manipulate CSS states?

The reason you get different behavior on the docs site when you hover with the mouse vs. applying a CSS hover state to one element in devtools is because there are multiple controls inside the Suggestions item with different hover styles (the suggestion item itself and Persona, in this case).

Since we have to use `forced-color-adjust: none`, we have more complex high contrast styles than usual since we need to ensure the inner Persona component does not override the SuggestionItem's `highlight`/`highlighttext` colors. When you hover with the mouse, both the Persona component and SuggestionItem component have their hover styles applied, which is when you get the correct behavior. You should also get the correct behavior if you don't use Persona at all, since SuggestionItem sets both the color and background-color to system color keywords.

If you are seeing a forced colors issue on your site, there is likely another component entirely that is overriding the SuggestionItem colors, but I can't be sure what's going on or debug it without a repro.

**Comment by vlumao** [comment]
Hi @smhigley, this issue happens when we just use PeoplePickerItemSuggestion. I wrote a simple code example in Codepen. Here is the link. https://codepen.io/vlumao/pen/wBaBQzd. I also took the screenshots.  Please let me know if you need more information.

![Image](https://github.com/user-attachments/assets/f6edc980-89cd-4318-9c23-6e5b055bfbe1)

![Image](https://github.com/user-attachments/assets/727fa18d-a480-4a39-bdd5-1c623d917e5c)

**Comment by smhigley** [comment]
@vlumao I'm going to go ahead and close this based on the 👍 on the last comment, but please feel free to re-open if there are any remaining issues that were unaddressed!
